ThunderCats/UITextField+TCAdditions.m

```diff
@@ -40,4 +40,27 @@ - (void)tc_setLeftAndRightPadding:(CGFloat)paddingWidth
     self.rightViewMode = UITextFieldViewModeUnlessEditing;
 }
 
+- (void)tc_addToolbarWithNextButton:(UITextField * __tc_nonnull)nextTextField
+                   toolbarColor:(UIColor * __tc_nonnull)toolbarColor
+{
+    [self tc_addToolbarWithButton:@"Next" toolbarColor:toolbarColor target:nextTextField action:@selector(becomeFirstResponder)];
+}
+
+- (void)tc_addToolbarWithButton:(NSString * __tc_nonnull)buttonTitle toolbarColor:(UIColor * __tc_nonnull)toolbarColor target:(id)target action:(SEL)selector
+{
+    UIBarButtonItem *button = [[UIBarButtonItem init] initWithTitle:buttonTitle style:UIBarButtonItemStylePlain target:target action:selector];
```

Comment:
There are a few instances in this code where `init` is being called directly on the class where `alloc` should be used instead. This will certainly cause a crash. I'd like to see some unit tests for these categories before before accepting this PR.